pyproject.toml

```diff
@@ -60,15 +59,14 @@ data = ['pooch>=1.6.0']
 developer = [
     'pre-commit',
     'ipython',
-    "tomli; python_version < '3.11'",
 ]
 docs = [
     'sphinx>=8.0',
     'sphinx-gallery[parallel]>=0.18',
     'numpydoc>=1.7',
     'sphinx-copybutton',
     'matplotlib>=3.7',
-    'dask[array]>=2022.9.2',
+    'dask[array]>=2022.11.0',
```

Comment:
Why this change? `dask[array]==2022.9.2` seems to work fine when I try to install it into a local Python 3.11 env?